beacon-chain/p2p/gossip_peer_controller.go

```diff
@@ -0,0 +1,192 @@
+package p2p
+
+import (
+	"context"
+	"math"
+	"sync"
+	"time"
+
+	"github.com/OffchainLabs/prysm/v7/beacon-chain/p2p/gossipcrawler"
+	"github.com/OffchainLabs/prysm/v7/cmd/beacon-chain/flags"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+const dialInterval = 1 * time.Second
+
```

Comment:
@nalepae per-topic-family peer dial policy for mesh/fanout is implemented in https://github.com/OffchainLabs/prysm/pull/16036/commits/451d2a8bc57e42d9f25ff457ed9b48a624498bb4. Let me know what you think.